changelog/fragments/replace-max-workers.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      Add "--max-concurrent-reconciles" flag to helm binary. Rename "max-workers" to "max-concurrent-reconciles"
+      in ansible/helm implementations.
```

Comment:
> Rename "max-workers" to "max-concurrent-reconciles" in ansible/helm implementations.

Is this referring to renaming/replacing the flag? In which case this is not an addition.
The verb "Rename" would be better suited to the `Changed` section.

Or is this referring to renaming the ansible controller Options field from `MaxWorkers` to `MaxConcurrentReconcilers`? I don't think that's a user facing change, and so we don't need to mention that in the CHANGELOG.


Also can you add a one line explanation for what `--max-concurrent-reconciles` does, or link to existing docs if we have those.

Reply:
`Add "--max-concurrent-reconciles" flag to helm binary` - meant the adding of a new flag, so that later we could deprecate `--max-workers`.
For `Rename "max-workers" to "max-concurrent-reconciles" in ansible/helm implementations.`, I meant `renaming the ansible controller Options field from MaxWorkers to MaxConcurrentReconcilers`. Ill remove this part from changelog, and modify it to `Changed`.